### ISSUE TITLE
Breaks with spaces in path

### ISSUE THREAD

**Issue by stefnotch** [CaptainUnbrauchbar/asp-language-support#9]
I placed my .lp files in a folder that has spaces in its path. Then I tried to compute all answer sets by right clicking and selecting the option.

Sadly this completely breaks this extension.

```
Microsoft Windows [Version 10.0.19044.2251]
(c) Microsoft Corporation. All rights reserved.

D:\Stefan\University\14_Semester 5\EWBS\exercise asp>c:\Users\Stefnotch\.vscode\extensions\ffrankreiter.answer-set-programming-language-support-0.6.0\clingo_win.exe d:\Stefan\University\14_Semester 5\EWBS\exercise asp\exercise1_0.lp 0
*** ERROR: (clingo): '5\EWBS\exercise': could not open input file!
*** Info : (clingo): Try '--help' for usage information

D:\Stefan\University\14_Semester 5\EWBS\exercise asp>
```

![image](https://user-images.githubusercontent.com/10220080/204157478-3505d508-9352-42d8-94c5-de23e1a53984.png)

**Comment by stefnotch** [comment]
Also, I'd like to take a moment to thank you very much for building this downright amazing extension. I really do enjoy the fact that with a more typical setup, it's simply "install extension - run clingo". No fiddling around with installing clingo, no having to figure out the correct command, instead it just works.

And since I really love things that just work in every case, I figured I'd open these issues to help you identify and fix the final few flaws.

**Comment by CaptainUnbrauchbar** [comment]
Fixed with next release 0.7.0 in a few minutes! Thanks for your kind words :)

**Comment by stefnotch** [comment]
Awesome, thank you for fixing this so exceedingly quickly! I really appreciate it
